docs/content/en/docs/concepts/overview/klc-cert-manager/_index.md

```diff
@@ -0,0 +1,19 @@
+---
+title: How it works
+icon: concepts
+layout: quickstart
+weight: 5
+hidechildren: true # this flag hides all sub-pages in the sidebar-multicard.html
+---
+
+### Keptn Cert Manager
+
+The Lifecycle Toolkit includes a Mutating Webhook which requires TLS certificates to be mounted as a volume in its pod. Since version 0.6.0 the certificate creation
```

Comment:
```suggestion
The Lifecycle Toolkit includes a Mutating Webhook which requires TLS certificates to be mounted as a volume in its pod. Since version 0.6.0, the certificate creation
```